src/builtins/Transformers.jl

```diff
@@ -177,6 +179,91 @@ MLJBase.docstring(::Type{<:FeatureSelector})      = "Filter features (columns) o
 MLJBase.load_path(::Type{<:FeatureSelector})      = "MLJModels.FeatureSelector"
 
 
+
+#### UNIVARIATE Discretizer ####
+"""
+UnivariateDiscretizer(n_classes=512)
+Returns a `MLJModels` for discretizing vectors of `Real`
+eltype, where `n_classes` describes the resolution of the
+discretization. Transformed vectors are of eltype `Int46`. The
+transformation is chosen so that the vector on which the transformer
```

Comment:
I really think we should make the output be a `CategoricalVector`, not an `Int` vector. So we are converting `Continuous` to `OrderedFactor`. 

Here's how you do that:

If `w` is the integer vector, return `categorical(w, ordered=true)`. For inverse transforming a categorical vector `w_cat`, you can first get the integer version using `w=MLJBase.int(w_cat)` and then apply the existing inverse transform to get the final continuous vector.